### DISCUSSION TITLE
Discussion about "Threema's business model" page

### DISCUSSION THREAD

**Discussion by pReya** [bitbetterde/Threematrix#2] [General]
Hello fellow community,

we just [published a little introduction](https://github.com/bitbatter/Threematrix/wiki/Legal-thoughts-and-Threema's-business-model) to the legal/license problems around a Threema-Matrix bridge.

We'd love to get input from the community. What are your thoughts? Let us know...

**Comment by rugk** [comment]
Good arguments/description there, however the one actual thing https://github.com/bitbetterde/Threematrix/issues/1 was about did not you actually handle, because you did not just ask Threema what they would suggest. I mean, as far as I said, they also allowed the other implementations, so they might as well just say you are okay with handling it as if you are a "usual Threema client"…

**Reply by pReya** [reply]
@rugk We will contact them, don't worry. As soon as we have anything to show regarding the Gateway API implementation.

The difference between the existing alternate clients (like OpenMittsu) and our bridge is, that our bridge enables easy automation scenarios. Therefore OpenMittsu is not competing with the paid Gateway API product. Our bridge potentially would do this. That's why we expect more backlash towards our project than towards something like OpenMittsu.
